src/IAM/Infrastructure/Persistence/MySQL/Logout/MySQLRefreshToken.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\Infrastructure\Persistence\MySQL\Logout;
```

Comment:
```suggestion
namespace Twitter\IAM\Infrastructure\Persistence\MySQL\Repository
```

See the [documentation](https://github.com/CPAprince/Twitter/blob/develop/docs/modular-monolith.md#project-structure)